CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+ * **2016-10-10**: [ORM] Add option to use custom Route class in ORM
```

Comment:
please also say that `cmf_routing.backend_type_orm` has been renamed to `cmf_routing.backend_type_orm_default`. somebody might have relied on this to be present, and it will be hard to discover this change when upgrading to 2.0